apps/makescene/makescene.cc

```diff
@@ -414,7 +422,7 @@ import_bundle_nvm (AppSettings const& conf)
         mve_cam.flen = mve_cam.flen / static_cast<float>(maxdim);
 
         mve::ByteImage::Ptr undist = mve::image::image_undistort_vsfm<uint8_t>
-            (std::dynamic_pointer_cast<mve::ByteImage const>(image), mve_cam.flen, nvm_cam.radial_distortion);
+            (std::dynamic_pointer_cast<mve::ByteImage const>(image), mve_cam.flen, cam_info.radial_distortion);
```

Comment:
Just for clarification: There must be no lines with more than 80 characters? (I'm asking this because the file contains lines written by other authors that are longer than 80 characters). No problem to make these adjustments, I just wanted to be sure.

Reply:
If so, I guess I touch only lines that are already modified by this PR?